OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -175,8 +175,8 @@ public interface INotifyDeployComplete
 
 	public interface INotifyDeployTriggered
 	{
-		void Deploy(Actor self, bool skipMakeAnim);
-		void Undeploy(Actor self, bool skipMakeAnim);
+		void Deploy(Actor self, bool skipMakeAnim, string[] bodyNames);
+		void Undeploy(Actor self, bool skipMakeAnim, string[] bodyNames);
```

Comment:
Thinking out loud here, but I hope this will lead to a solid implementation: it would be nice if we could simplify these down to `void (Un)Deploy(Actor self, string deployType)` and then have the skip/bodies done as a property of the trait that is being triggered.